### PR TITLE
Add sanitizing html special characters

### PR DESCRIPTION
**Pull request by zamronypj**
Hello!

* Type: new feature
* Link to issue:

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I wrote some tests for this PR.

Small description of change:

Add ability to sanitize special characters such < >

Thanks

### CODE REVIEW

**Review by sergeyklay — APPROVED** [review]

**Comment by sergeyklay** [comment]
I will keep this open for a day or two to allow other contributors a chance to speak up, and to take a fresh look at it later. But at the moment it seems good to me.

Cc: @phalcon/collaborator @phalcon/framework-team

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # [3.2.0](https://github.com/phalcon/cphalcon/releases/tag/v3.2.0) (2017-XX-XX)
+- Added ability to sanitize special characters to `Phalcon\Filter`
```
Comment (Jurigag) [inline CHANGELOG.md]: It should be added at the end of changelist for this version.

**Comment by Jurigag** [comment]
Test is failing

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**